src/app/tasks/tasks-list/task-item/task-item.component.html

```diff
@@ -1,14 +1,15 @@
-<li [id]="task.id">
-  <input
-    [id]="'checkbox' + task.id"
-    type="checkbox"
-    [checked]="task.isDone"
-    (click)="markAsDone()"
-  />
+<li [id]="task.id" [appDrag]="{onDrag: deleteTask.bind(this), onClick: showDetails.bind(this)}">
```

Comment:
Please define the object of appDrag inside the `ts` file and use it in the `html`.
Please also use arrow function instead of `bind` function